site-templates/pages/v3-tips-and-tricks.gomd

```diff
@@ -118,5 +125,4 @@ And the answer is this, with the details of the specified type:
     }
 }
 </pre>
-
 </p>
```

Comment:
@NoahBluescape , please add a newline at the end of the file.

Reply:
Done, and checked the other files and added them as needed.